v63003/u_inref/gtm8847.csh

```diff
@@ -0,0 +1,29 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+#
+#
+echo "# Attempting to overflow the string pool"
+$ydb_dist/mumps -run test2^gtm8847 >>& errors.outx
+set nonomatch;rm core*
```

Comment:
Add a comment that "rm core*" is needed to remove corefile created by STPOFLOW error. Why is the nonomatch needed?